test/e2e/prune_test.go

```diff
@@ -59,6 +59,18 @@ var _ = Describe("Podman prune", func() {
 		Expect(podmanTest.NumberOfContainers()).To(Equal(1))
 	})
 
+	It("podman container prune after create containers", func() {
+		stop := podmanTest.Podman([]string{"create", "--name", "test", BB})
+		stop.WaitWithDefaultTimeout()
+		Expect(stop.ExitCode()).To(Equal(0))
+
+		prune := podmanTest.Podman([]string{"container", "prune", "-f"})
```

Comment:
Can you add a container that's in the "Configured"  state to this test to make sure it's removed?

Reply:
after `podman create` the container status is `Configured` https://bugzilla.redhat.com/show_bug.cgi?id=1810874#c7

Reply:
OK.  My main ask is we're now pruning containers that are in the Created and Configured state.  Can you set up at least one of each and make sure they get pruned?  Currently, it looks like we're only testing Configured  then.

Reply:
Do you know how to create the container has status `Created`?
```
$ podman create --name ctr alpine
bac6adb052817e1a9ea88966753b8a4c6f389e6b5116c17f33ec22475c48d302
$ podman inspect ctr --format {{.State.Status}}
configured
```
```
$ podman run --name ctr --entrypoint /error alpine
Error: container_linux.go:346: starting container process caused "exec: \"/error\": stat /error: no such file or directory": OCI runtime command not found error
c0e217d5c193  docker.io/library/alpine:latest  /bin/sh  3 seconds ago  Created         charming_booth
$ podman inspect ctr --format {{.State.Status}}
configured
```